Assets/Scripts/Utils/Math.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+
+namespace Utils
+{
+    public static class MathUtils
+    {
+        /// <summary>
+        /// P(t) = (K * Po * e^(r*t)) / (Po * e^(r*t) + K + Po)
+        /// </summary>
+        /// <param name="tick">t, entier positif dans l'interval [0, max double]</param>
+        /// <param name="popMin">Po, entier positif dans l'interval [1, max long]</param>
+        /// <param name="capaciteMax">K, entier positif dans l'interval [popMin+1, max long]</param>
+        /// <param name="croissance">r, taux de croissance dans l'interval [0.01, 1]</param>
+        /// <returns>P(t) soit la population au tick t. [popMin, capaciteMax]</returns>
+        public static double FonctionLogistique(double tick, long popMin, long capaciteMax, float croissance)
+        {
```

Comment:
Plz change this for a function from this website: https://easings.net/#easeInOutCubic